### ISSUE TITLE
When showImages is turned off in user settings, inline image metadata is lost

### ISSUE THREAD

**Issue by Half-Shot** [vector-im/element-web#16935]
(I'm failing to find the right title phrasing here, so update if you have a better one)

I have showImages turned off on my account, which causes inline images to be discarded (expected). However this means that if someone sends me a message with an inline image that conveys a meaning, all context is lost there. I'd probably expect to see the use of the `alt` tag in it's place at a minimum, just so the context isn't lost.

Related (and would probably help my issue https://github.com/vector-im/element-web/issues/16934)

**Comment by turt2live** [comment]
ftr this is a known deficiency given inline images is more of an accidental feature than a supported one

**Comment by Half-Shot** [comment]
Yep, especially given Element Web has no way to create these yet but felt it was worth creating a bug report so the next user is aware that it's a known problem.
